tests/unittests/netmap/topology_layer2_testcase.py

```diff
@@ -1,3 +1,5 @@
+# ruff: noqa: F841
```

Comment:
When adding noqa comments, I think it would be nice if they were added as separate commits, so the commit log can explain why this is needed.